README.rst

```diff
@@ -17,3 +17,19 @@ and development purposes you can use `message_factory(...)` which allows you to
     message_factory(t_name=<str>, message_contents=<Tuple[str, str]>, [num_partitions=<int>], [replication_factor=<int>])
 
 bracketed arguments are optional.
+
+Development and Testing
+#######################
+For development and testing it is very useful to have a local instance of Kafka running on your machine
+
+See here for simple docker compose to get your own Kafka server up and running quickly.
+`Single Zookeeper/Multiple Kafka<https://github.com/simplesteph/kafka-stack-docker-compose#single-zookeeper--multiple-kafka`
```

Comment:
It might be a good idea to copy the docker-compose to our repo (with link/credits to the original place). The Internet is a wild place...